src/Components/ADT3DSceneBuilder/Internal/Behaviors/VisualRules/VisualRuleList.tsx

```diff
@@ -103,13 +115,37 @@ function getListItems(
         ];
     };
 
+    function getIconStart(item: IVisualRule) {
+        const [meshCount, badgeCount] = getBadgesAndMeshesCount(item);
+        let icon;
+        if (meshCount && badgeCount) {
+            icon = () => (
+                <Svg
```

Comment:
This could be a Fluent Icon or FontIcon, why are we using this new external component?

Reply:
Played around with this code a bit, and tried to do this with Fluent. This seems like a better solution and simpler to maintain.